x-pack/plugins/cases/public/components/configure_cases/mapping.tsx

```diff
@@ -27,21 +27,30 @@ const MappingComponent: React.FC<MappingProps> = ({
   isLoading,
   mappings,
 }) => {
-  const selectedConnector = useMemo(() => connectorsConfiguration[connectorActionTypeId], [
-    connectorActionTypeId,
-  ]);
+  const { triggersActionsUi } = useKibana().services;
+  const selectedConnector = useMemo(
+    () => triggersActionsUi.actionTypeRegistry.get(connectorActionTypeId),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [connectorActionTypeId]
```

Comment:
What will happen if we include `triggersActionsUi` in the deps?

Reply:
It doesn't seem to affect it somehow. Fixed it.